playerstats.py

```diff
@@ -36,13 +36,15 @@ def getPlayerStats(club):
     df_topscorers = pd.DataFrame.from_dict(top_scorer)
     return df_topscorers
 
+
 def getGoalsScored(club):
     df_topscorers = getPlayerStats(club).sort_values(by=['goals'], ascending=False, ignore_index=True).head(5)
-    df_topscorers.index = df_topscorers.index+1
-    table=tabulate(df_topscorers,tablefmt='plain',colalign=['left','left'],showindex=False)
+    df_topscorers.index = df_topscorers.index + 1
+    table = tabulate(df_topscorers, tablefmt='plain', colalign=['left', 'left'], showindex=False)
     return table
 
+
 def main(msgLower):
     match msgLower:
-      case '!goals':
-          return getGoalsScored('Arsenal')
+        case '!goals':
```

Comment:
I would suggest not to change the tab spaces

Reply:
I didn't. The second commit was to restore the tab space, which was changed by mistake